tests/utils/trade.py

```diff
@@ -111,6 +111,15 @@ def get_order(self, robot_index=1, first_encounter=False):
         headers = self.get_robot_auth(robot_index, first_encounter)
         self.response = self.client.get(path + params, **headers)
 
+    def get_notifications(self, created_at=0, robot_index=1, first_encounter=False):
+        """
+        Fetch the latest state of the order
+        """
+        path = reverse("notifications")
+        params = f"?created_at={created_at}"
+        headers = self.get_robot_auth(robot_index, first_encounter)
+        self.response = self.client.get(path + params, **headers)
```

Comment:
During tests, `trade.response` is always the return of the latest GET to `/api/order`. Maybe here we want to create a new variable on the Trade class `notification` . This way, in tests we evaluate the return of `trade.response` and of `trade.notification` and don't get confused when improving them in the future. 

I would therefore edit this one as:
```
self.notification = self.client.get(path + params, **headers)
```
`notification` or `last_notification` , the one name you prefer!

Reply:
I ended up moving it out of Trade, I think it didn't make sense to have it there